alternator/executor.cc

```diff
@@ -2812,6 +2812,7 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
     }
 
     _stats.api_operations.batch_write_item_batch_total += batch_size;
+    _stats.api_operations.batch_write_item_count.add(batch_size);
```

Comment:
I feel like an idiot continuing to bikeshed this patch, but...
If you see one variable called batch_write_batch_total and one called batch_write_item_count you'll assume the second counts the operations!
I though you were going to rename it something with histogram. How about batch_write_item_items_histogram or batch_write_item_count_histogram?

In any case, I see below you called the metric batch_item_count_histogram. I think it's unnecessary and confusing when the variable's name is very different from the metrics name.

Reply:
sure, easy change